app/views/providers/confirm_dwp_non_passported_applications/show.html.erb

```diff
@@ -12,26 +12,37 @@
         form:,
       ) do %>
 
-    <div class="interruption-panel govuk-!-margin-bottom-6">
-      <h1 class="govuk-heading-l"><%= t(".title_html") %></h1>
+    <div class="moj-interruption-card govuk-!-margin-bottom-6">
+      <div class="moj-interruption-card__content">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
```

Comment:
Is the grid required? If yes, would it be ideal to have it outside the interruption card component?